spec/values/calculation/abs.hrx

```diff
@@ -0,0 +1,182 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `abs()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> zero/input.scss
+a {b: abs(0)}
+
+<===> zero/output.css
+a {
+  b: 0;
+}
+
+<===>
+================================================================================
+<===> positive/input.scss
+a {b: abs(1)}
+
+<===> positive/output.css
+a {
+  b: 1;
+}
+
+<===>
+================================================================================
+<===> negative/input.scss
+a {b: abs(-5.6)}
+
+<===> negative/output.css
+a {
+  b: 5.6;
+}
+
+<===>
+================================================================================
+<===> simplification/input.scss
+a {
+  b: abs(1px + 2px - var(--c))
+}
+
+<===> simplification/output.css
+a {
+  b: abs(3px - var(--c));
+}
+
+<===>
+================================================================================
+<===> operation/preserves_units/input.scss
```

Comment:
Not a 100% sure if we should keep this test, I don't think the simplification test covers this, right?